src/controllers/orderStatus.controller.js

```diff
@@ -31,4 +30,16 @@ const trackOrderStatus = async (req, res) => {
   }
 };
 
-export default trackOrderStatus;
+export const getOrders = async (req, res) => {
+  const buyerId = req.user.id;
+  const role = req.user.role;
+  const allOrders = await Orders.findAll({ where: { userId: buyerId } });
```

Comment:
You should create a helper function to do all of this cz don't belong to controllers

Reply:
@ba-lambert Above codes have been updated

Reply:
This should be created outside controllers